decisions/mnm_society_decisions.txt

```diff
@@ -1532,50 +1544,124 @@ society_decisions = {
 		}
 
 		effect = {
-			tooltip = {
-				add_trait = celibate
-			}
-			character_event = {
-				id = MNM.4090
-			}
+			change_society_currency = 150
+			add_trait = celibate
+			add_character_modifier = { name = keeping_vows_purity years = 10 }
+			
+			# tooltip = {
+				# add_trait = celibate
+			# }
+			# character_event = {
+				# id = MNM.4090
+			# }
 		}
 		
 		ai_will_do = {
-			factor = 0
+			factor = 0.25
+			modifier = {
+				factor = 0
+				is_married = yes
+			}
+			modifier = {
+				factor = 2
+				trait = chaste
+			}
+			modifier = {
+				factor = 0.3
+				trait = lustful
+			}
 		}
 	}
 
 	monastic_orders_stop_celibate = {
-		ai = no
+		# Warcraft
+		is_high_prio = yes
+
+		ai_check_interval = 24
+
+		only_playable = yes
 
 		potential = {
-			OR = {
-				is_christian_monastic_member_trigger = yes
-				is_dharmic_monastic_member_trigger = yes
+			is_adult = yes
+			is_incapable = no
+			prisoner = no
+			society_rank > 1
+			society_member_of = sisterhood_of_elune
+			trait = celibate
+			
+			# OR = {
```

Comment:
I don't know if I fully understand. The code was written then immediately commented out? Why write in first place?